admin_manual/configuration_server/activity_configuration.rst

```diff
@@ -0,0 +1,27 @@
+Configuring the Activity App
+============================
+
+You can configure your ownCloud server to automatically send out e-mail notifications
+to your users for various events like:
```

Comment:
If the admin has not configured the e-mail setup no e-mail notifications will be sent out. So the main part needs to be done by the admin.